orc8r/cloud/go/services/tenants/servicers/internal_servicer/servicer.go

```diff
@@ -11,7 +11,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package servicers
+package internal_servicer
```

Comment:
package and directory names should be "internal_servicers" instead of "internal_servicer"